wagtail_airtable/templates/wagtailsnippets/snippets/type_index.html

```diff
@@ -0,0 +1,51 @@
+{% extends "wagtailsnippets/snippets/type_index.html" %}
+{% load i18n wagtailadmin_tags wagtail_airtable_tags %}
+
+
+{% block content %}
+    {% can_import_model model_opts.label as can_import_model %}
```

Comment:
I feel like we really need to refactor the Wagtail templates in order to make adding header buttons much simpler - I've encountered having to do this whole override too many times!

Reply:
Glad you brought this up. Yes, I agree. This might be something we bring up in a slightly larger Wagtail convo. This isn't the first time I've had to copy/paste an entire template just to add a few extra lines. 

I'll mark as resolved and merge in this PR. But we should definitely find a fix for these Wagtail admin templates.